src/onpage.js

```diff
@@ -34,29 +34,21 @@
 	},
 	[
 		'addListener',
-		'applyCode',
-		'autoAppIndex',
 		'banner',
 		'closeBanner',
 		'closeJourney',
-		'creditHistory',
 		'credits',
```

Comment:
Is this `credits` string still needed?

Reply:
Looks deleted actually https://github.com/BranchMetrics/web-branch-deep-linking-attribution/pull/926/files#diff-cf134fe4c8883dd92eef5f2b91797d08dd1ffed8e57ee3c39e794bdf93e3940dL1584

Reply:
good catch. Not needed.